kadai1/oamam/imgconv/imgconv.go

```diff
@@ -0,0 +1,125 @@
+package imgconv
+
+import (
+	"errors"
+	"image"
+	"image/gif"
+	"image/jpeg"
+	"image/png"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type ext string
+
+const (
+	GIF  ext = "gif"
+	PNG  ext = "png"
+	JPG  ext = "jpg"
+	JPEG ext = "jpeg"
+)
+
+var extensions = []ext{GIF, PNG, JPG, JPEG}
+
+type target struct {
+	inputPath  string
+	inputExt   ext
+	fileName   string
+	outputPath string
+	outputExt  ext
+}
+
+func (t *target) GetInputFile() string {
```

Comment:
外部パッケージから参照しないのであれば、小文字で大丈夫かと！
```suggestion
func (t *target) getInputFile() string {
```